src/common/support_utils.h

```diff
@@ -0,0 +1,25 @@
+#ifndef _UTILS_H
+#define _UTILS_H
```

Comment:
filename changed, but the preprocessor guard was left intact. This may cause hard-to-find build errors
use `#pragma once`